docs/maintainers/repository-structure.md

```diff
@@ -2,7 +2,7 @@
 
 This repository can be viewed both via GitHub and via the v3 website. This page describes the repository as found on GitHub. It is roughly equivalent to the website, with filenames being titelized.
```

Comment:
`titelized` is not a word. What shall I change it to? What's it even trying to say?

Reply:
I think it is referring to a built-in Ruby method: https://apidock.com/rails/String/titleize

> Capitalizes all the words and replaces some characters in the string to create a nicer looking title.

Maybe we should indeed rephrase.